docs/02-running/01-running-locally.md

```diff
@@ -3,32 +3,53 @@
 By now you should have:
 - installed all software dependencies
 - created an AWS profile called `media-service`
-- created an AWS CloudFormation stack
-- setup nginx proxies
-- generated configuration files in `/etc/gu`
+- run `setup.sh`
 
-We can start Grid by using the [dev-start.sh](../../dev-start.sh) script:
+## `start.sh`
+We can start Grid by using the [start.sh](../../dev/script/start.sh) script.
+
+From the project root:
 
 ```shell script
-./dev-start.sh
+./dev/script/start.sh
 ```
 
-The following flags are available for `dev-start.sh`:
-- `--debug` which will make port `5005` available for [debugging in IntelliJ](https://www.jetbrains.com/help/idea/attaching-to-local-process.html)
-- `--ship-logs` which will ship logs to a [local elk](https://github.com/guardian/local-elk) stack over tcp on port `5000`
+This process takes a little while to complete.
+
+### Available flags
+There are a few options available for `start.sh`.
+
+#### `--debug`
+Adding the `--debug` flag which will make port `5005` available for [debugging in IntelliJ](https://www.jetbrains.com/help/idea/attaching-to-local-process.html)
+or your favourite IDE.
+
+#### `--ship-logs`
+Adding the `--ship-logs` flag will ship logs to a [local elk](https://github.com/guardian/local-elk) stack over tcp on port `5000`.
+You'll need to be sure `local-elk` is running beforehand to sure port `5000` is open.
+
+#### `--with-local-auth`
+If you previously ran `setup.sh` using this flag, you'll also need to use it in `start.sh`.
 
 ## `the_pingler.sh`
 Grid consists of many micro-services. Some services don't compile until you hit them for the first time.
-There is a script included that will do this for you called [the_pingler.sh](../the_pingler.sh).
-{
+There is a script included that will do this for you called [the_pingler.sh](../../dev/script/the_pingler.sh).
+
 This is a simple shell script that keeps pinging the healthcheck endpoints of the various
 services and reports it via the colour of the URL.  This is needed because some services do
 not start to function correctly until they have been contacted at least once.
 It's recommended to keep this running in the background while you have the stack started up
 and keep it running in a background terminal.
 
-```bash
-./the_pingler.sh
+```shell script
+./dev/script/the_pingler.sh
 ```
 
-If everything has worked, we should be able to access Grid in a browser on `https://media.local.dev-gutools.co.uk`.
+## Accessing Grid
+If everything has worked, we should be able to access Grid in a browser on `https://media.local.dev-gutools.co.uk`
+where we'll begin an authentication flow.
+
+If you've used the `--with-local-auth` flag you'll be directed to `localhost:9014` for authentication.
```

Comment:
This isn't a `.local.dev-gutools.co.uk` domain as I couldn't get `oidc-provider` working with it properly. I suspect this might be useful - https://github.com/panva/node-oidc-provider/blob/v5.x/docs/configuration.md#trusting-tls-offloading-proxies.

Using `localhost:9014` doesn't cause any issues and does reiterate that this is running locally and not PROD ready, which is a good thing. (I'm grasping here! 😅)